### PR TITLE
Rewrite of Key concepts page, fixes #566

### LINKED ISSUE #566: Rewrite page - Key concepts
With the updates to the schema for BODS 0.4 to incorporate the record concept, the [Key concepts page](https://standard.openownership.org/en/0.3.0/schema/concepts.html) needs a wholesale rewrite.

This can also mop up https://github.com/openownership/data-standard/issues/455.

### PR DESCRIPTION
**Pull request by kd-ods**
# Overview

This is currenty WIP until the docs build and I can see the page. (I think the image display sizes will need to change.)

- What does this pull request do?
Edit of the concepts.rst page, with new diagrams, adds a record-ids.rst stub page.

- How can a reviewer test or examine your changes?
Build the docs. Read the page, suggest edits. 

- Who is best placed to review it?

Kathryn

(Closes/Relates to) issue: #566 

## Translations

- [ ] New or edited strings appearing as a result of this PR will be picked up for translation
- [ ] I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

- [ ] I've checked that the documentation builds without failing. See: https://github.com/openownership/data-standard#building-the-documentation-locally
- [ ] I've thought about how and when this needs to be released. See:
      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- [ ] I've updated the changelog, if necessary.
- [ ] I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- [ ] I've added or edited any other related documentation. See: https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md

### CODE REVIEW

**Comment by kd-ods** [comment]
@kathryn-ods - this is ready for your review.

If you have an issue with the navigation menu overlapping the main content area - ignore. I'll spin up a separate issue for that.

File: docs/schema/concepts.rst [hunk 1]
```diff
@@ -5,112 +5,97 @@ Key concepts
...
 
+Representing beneficial owners
+------------------------------
 
-Ownership or control through arrangements
-------------------------------------------
+In a BODS dataset, the fact that a natural person is a beneficial owner of an entity is represented by including this information in the Relationship statement linking the two.
 
-In some cases ownership or control is exercised through:
+It is possible to represent in BODS datasets an entity's declaration that it has no beneficial owners (according to a jurisdiction's definition of a beneficial owner).
```
Comment (kathryn-ods) [inline docs/schema/concepts.rst:90]: > It is possible to represent in BODS datasets an entity's declaration that it has no beneficial owners (according to a jurisdiction's definition of a beneficial owner).

Maybe cut 'in BODS datasets' or I think it should have commas either side as it kind of breaks up the sentence

**Review by kathryn-ods — APPROVED** [review]
Looking good! Couple of small thoughts

File: docs/schema/concepts.rst [hunk 2]
```diff
@@ -5,112 +5,97 @@ Key concepts
...
-* Entities (including companies, trusts and arrangements)
-* Natural persons (who own or control entities)
 
-Statements
------------
+Beneficial ownership concepts
+-----------------------------
 
-Details of how an interested party controls or owns a company or other legal entity are wrapped in an ownership-or-control statement.
+As outlined in the :ref:`Primer <what-is-bo>`, a natural **person** is a **beneficial owner** of an **entity** because of certain **interests**. These interests may be rooted in legal ownership, or come from controlling the entity, or using its assets. The **relationship** between the beneficial owner and the entity may be **direct**, **indirect** or both. Where it is indirect, **intermediary** entities, people and their relationships, are part of the **beneficial ownership network**.
```
[the remark this answers is not in the document]
Comment (kd-ods) [reply docs/schema/concepts.rst:14]: @kathryn-ods - yeah, sometimes lawyers act as - e.g. - nominees and are called 'intermediaries'. In BODS we wouldn't represent them as an intermediate node though. So this is an example of where there isn't a clean mapping between the conceptual model and the BODS data model. Which is fine, as long as we're clear about that.

[the remark this answers is not in the document]
Comment (kd-ods) [reply docs/schema/concepts.rst:14]: Deleted 'As outlined...'

File: docs/schema/concepts.rst (hunk 1 above)
Comment (kd-ods) [reply docs/schema/concepts.rst:90]: Done

**Comment by kd-ods** [comment]
@kathryn-ods - I just need a quick re-review after I made edits.

**Review by kathryn-ods — APPROVED** [review]

**Pull request merged**